File: pgm/ABC_Java/ABC_447_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        final int one = 1;
        final int two = 2;
        final int zero = 0;
        int N = Integer.parseInt (arr[zero]);
        int M = Integer.parseInt (arr[one]);
        System.out.println (M <= (N + one) / two ? Yes : No);
        System.exit (0);
    }
}

File: pgm/ABC_C#/ABC_447_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const string Yes = "Yes";
        const string No = "No";
        const int one = 1;
        const int two = 2;
        const int zero = 0;
        int N = int.Parse(arr[zero]);
        int M = int.Parse(arr[one]);
        Console.WriteLine(M <= (N + one) / two ? Yes : No);
        Environment.Exit(0);
    }
}